spec/features/who_is_on_duty_spec.rb

```diff
@@ -0,0 +1,168 @@
+          expect(page.body).to have_content('On Duty Tonight')
+          expect(page.body).to have_selector('#out-of-hours', text: 'Stuart Munro')
+        end
+
+        scenario 'What time does the rota switch to out-of-hours?' do
+          expect(page.body).to have_content('Starts at 17:00')
+        end
+
+      end
+
+    end
+
+    feature 'It is out of hours' do
+
+      before do
+        Timecop.freeze(Time.local(2015, 9, 2, 20)) do
+          reset_roster!
+          basic_auth
+          visit '/admin'
+        end
+      end
+
+      context '(these come from pagerduty)' do
+
+        scenario 'so there is no list of who is on duty tonight' do
+          expect(page.body).not_to have_content('On Duty Tonight')
+        end
+
+        scenario 'and the daywatch is not shown' do
+          the_team.each do |member|
+            expect(page.body).not_to have_content(member)
+          end
+        end
+
+        # TODO: IRPagerduty needs some work to work out the exact format
+        # of the JSON the API returns. For now, this is sufficient as it
+        # has been manually tested with the real feed (shouldn't show Stuart
+        # twice).
+        scenario 'Which pair of people are on duty right now?' do
+          expect(page.body).to have_content('Stuart Munro', count: 2)
+        end
+
+        scenario 'What time does the rota switch to in hours support?' do
+          expect(page.body).to have_content('Ends at 22:59')
+        end
+
+      end
+
+    end
+
+    feature 'It is a weekend and therefor out-of-hours' do
+
+      before do
+        Timecop.freeze(Time.local(2015, 9, 5, 13)) do
+          reset_roster!
+          basic_auth
+          visit '/admin'
+        end
+      end
+
+      scenario 'so there is no list of who is on duty tonight' do
+        expect(page.body).not_to have_content('On Duty Tonight')
+      end
+
+      scenario 'and the daywatch is not shown' do
+        the_team.each do |member|
+          expect(page.body).not_to have_content(member)
+        end
+      end
+
+      scenario 'Which pair of people are on duty right now?' do
+        expect(page.body).to have_content('Stuart Munro', count: 2)
+      end
+
+      scenario 'What time does the rota switch to in hours support?' do
+        expect(page.body).to have_content('Ends at 22:59')
+      end
+
+    end
+
+    feature 'It is in the no-support period' do
+
+      before do
+        Timecop.freeze(Time.local(2015, 9, 3, 23)) do
+          reset_roster!
+          basic_auth
+          visit '/admin'
+        end
+      end
+
+      scenario 'it tells me there is no support' do
+        expect(page.body).to have_content('Support hours resume at 10:01')
+      end
+
+    end
+  end
+
+  context 'I am not signed in' do
+    before do
+      Timecop.freeze(Time.local(2015, 9, 2, 20)) do
+        reset_roster!
+        visit '/'
+      end
+    end
+
+    feature 'At any time' do
```

Comment:
Clearer now?  First time I've really used features in rspec and I was finding it a bit tricky to get the intent clear enough.